src/pages/AddCertificate/AddCertificate.tsx

```diff
@@ -0,0 +1,248 @@
+    }
+  };
+
+  const handleReset = () => {
+    setCertificateData({
+      supplier: '',
+      certificateType: '',
+      validFrom: null,
+      validTo: null,
+    });
+    setPdfPreview(null);
+  };
+
+  const handlePdfUpload = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const file = e.target.files?.[0];
+    if (file) {
+      const reader = new FileReader();
+      reader.onload = () => {
+        setPdfPreview(reader.result as string);
+      };
+      reader.readAsDataURL(file);
+    }
+  };
+
+  return (
+    <div className="certificate-form-container">
+      <div className="left-side">
+        <div className="row">
+          <label htmlFor="supplier">Supplier</label>
+          <div className="input-wrapper">
+            <input
+              type="text"
+              id="supplier"
+              name="supplier"
+              value={certificateData.supplier}
+              onChange={handleInput}
+              className="supplier-input"
+            />
+
+            <button className="search-button">
+              <SearchIcon />
+            </button>
+
+            <button className="clear-button">
+              <CrossIcon />
+            </button>
+          </div>
+        </div>
+        {errors.supplier && <p className="error-message">{errors.supplier}</p>}
+
+        <div className="row">
+          <label htmlFor="certificate-type">Certificate type</label>
+          <select
+            id="certificate-type"
+            name="certificateType"
+            value={certificateData.certificateType}
+            onChange={handleInput}
+            defaultValue=""
+          >
+            <option
+              value=""
+              disabled
+            >
+              Select your option
+            </option>
+            <option value="Permission of Printing">
+              Permission of Printing
+            </option>
+            <option value="OHSAS 18001">OHSAS 18001</option>
+          </select>
+        </div>
+        {errors.certificateType && (
+          <p className="error-message">{errors.certificateType}</p>
+        )}
+
+        <CustomDateInput
+          id="start-date"
+          label="Valid from"
+          name="validFrom"
+          value={certificateData.validFrom}
+          onChange={(date) => handleDateChange('validFrom', date)}
+        />
+        {errors.validFrom && (
+          <p className="error-message">{errors.validFrom}</p>
+        )}
+
+        <CustomDateInput
+          id="end-date"
+          label="Valid to"
+          name="validTo"
+          value={certificateData.validTo}
+          onChange={(date) => handleDateChange('validTo', date)}
+        />
+        {errors.validTo && <p className="error-message">{errors.validTo}</p>}
+        {errors.validRange && (
+          <p className="error-message">{errors.validRange}</p>
+        )}
+      </div>
+      <div className="right-side">
+        <div>
+          <label className="upload-button">
```

Comment:
wrap this to component as well, and this component can go in common - components